### PR TITLE
Renames Models, Model Schemas - round 2

### PR DESCRIPTION
**Pull request by diana-lakatos**
Round 2 of renaming, this should cover everything that round 1 didn't cover (Get Started, Use Cases, Best Practices). 
Intentionally left out the User Research and Release Notes sections, so that the history of what was happening is not overwritten. 

@Slashek Please review! Thank you! 

Part of #1332

### CODE REVIEW

File: app/views/pages/api-reference/graphql/glossary.liquid [hunk 1]
```diff
@@ -25,11 +25,11 @@ An alternative name given to the result of a field to avoid conflicts during dat
 
 ```graphql
 {
-  notes: models(per_page: 20, filter: { name: { value: "notes" } }) {
+  notes: records(per_page: 20, filter: { name: { value: "notes" } }) {
     results { id }
   }
 
-  cars: models(per_page: 20, filter: { name: { value: "cars"} }) {
+  cars: records(per_page: 20, filter: { name: { value: "cars"} }) {
```
Comment (Slashek) [inline app/views/pages/api-reference/graphql/glossary.liquid]: @diana-lakatos `name` has to be changed to `table` here

File: app/views/pages/api-reference/graphql/glossary.liquid [hunk 2]
```diff
@@ -25,11 +25,11 @@ An alternative name given to the result of a field to avoid conflicts during dat
 
 ```graphql
 {
-  notes: models(per_page: 20, filter: { name: { value: "notes" } }) {
+  notes: records(per_page: 20, filter: { name: { value: "notes" } }) {
```
Comment (Slashek) [inline app/views/pages/api-reference/graphql/glossary.liquid]:
```suggestion
  notes: records(per_page: 20, filter: { table: { value: "notes" } }) {
```

File: app/views/pages/api-reference/graphql/glossary.liquid (hunk 1 above)
Comment (Slashek) [inline app/views/pages/api-reference/graphql/glossary.liquid]:
```suggestion
  cars: records(per_page: 20, filter: { table: { value: "cars"} }) {
```

File: app/views/pages/best-practices/amp/feeding-data-amp-pages-graphql.liquid [hunk 3]
```diff
@@ -112,7 +112,7 @@ Create the GraphQL query that fetches product data.
 
 ```graphql
 query get_products($slug: String) {
-  products: models(
+  products: records(
     filter: {
       name: { value: "product" }
```
Comment (Slashek) [inline app/views/pages/best-practices/amp/feeding-data-amp-pages-graphql.liquid]:
```suggestion
      table: { value: "product" }
```

File: app/views/pages/best-practices/gdpr/gdpr-compliance.liquid [hunk 4]
```diff
@@ -183,7 +183,7 @@ query get_newsletter(
   $email: String
 )
 {
-  models (
+  records (
     filter: {
       name: { exact: "newsletter" }
```
Comment (Slashek) [inline app/views/pages/best-practices/gdpr/gdpr-compliance.liquid]:
```suggestion
      table: { value: "newsletter" }
```

File: app/views/pages/best-practices/seo/creating-seo-friendly-urls.liquid [hunk 5]
```diff
@@ -66,7 +66,7 @@ properties:
 Create a GraphQL query with country and city.
 ```graphql
 query get_catalogue_listings($country: String, $city: String) {
-  models(
+  records(
     per_page: 20,
     filter: {
       name: {
```
Comment (Slashek) [inline app/views/pages/best-practices/seo/creating-seo-friendly-urls.liquid]:
```suggestion
      table: {
```

File: app/views/pages/developer-guide/blog/blog-settings-posts.liquid [hunk 6]
```diff
@@ -29,7 +29,7 @@ You can fetch all post details with:
 
 ```graphql
 query get_blog_posts($per_page: Int = 20, $page: Int, $title: String, $id: ID, $slug: String, $published_at_lte: String, $published_at_lt: String, $published_at_gt: String, $published_at_gte: String, $tags: [String!], $blog_instance_id: String) {
-  blog_posts: models(
+  blog_posts: records(
```
Comment (Slashek) [inline app/views/pages/developer-guide/blog/blog-settings-posts.liquid:32]: here `name` in the `filter` also has to be changed to `table`

File: app/views/pages/developer-guide/data-import-export/import.liquid [hunk 7]
```diff
@@ -227,19 +227,19 @@ pos-cli data import --zip --path=archive.zip staging-instance
...
 
-- Before import, your model schemas need to be deployed.
+- Before import, your Tables need to be deployed.
 - CSV files only support comma separated values, if in doubt, it might be a good idea to use a linter like [CSVlint](https://github.com/Clever/csvlint) to check the file.
 - A well formatted file has a header with names of columns available for the given type, and one or more rows of data.
 
 ### Example file for Records
 
 ```csv
-id,user_id,created_at,updated_at,properties,model_schema
```
Comment (Slashek) [inline app/views/pages/developer-guide/data-import-export/import.liquid]: I'm afraid we are not able to change it yet, but will log in a ticket

File: app/views/pages/developer-guide/modules/creating-module.liquid [hunk 8]
```diff
@@ -43,11 +43,11 @@ admincms
 
 Create a GraphQL query that pulls a list of Records.
 
-##### modules/admincms/private/graphql/get_models.graphql
+##### modules/admincms/private/graphql/get_records.graphql
 
 ```graphql
-query get_models {
-  models(
+query get_records {
```
Comment (Slashek) [inline app/views/pages/developer-guide/modules/creating-module.liquid:49]: I think this query won't run, `human_name` should be `table`

File: app/views/pages/developer-guide/pos-cli/developing-graphql-queries-using-pos-cli-gui.liquid [hunk 9]
```diff
@@ -89,8 +89,8 @@ When your query or mutation is accepting arguments, you can set default values f
 Example of using default argument values:
 
 ```graphql
-query get_models($name: String = "feedback") {
-  models(
+query get_records($name: String = "feedback") {
+  records(
     per_page: 10
     filter: {
       name: { value: $name }
```
Comment (Slashek) [inline app/views/pages/developer-guide/pos-cli/developing-graphql-queries-using-pos-cli-gui.liquid]:
```suggestion
      table: { value: $name }
```

File: app/views/pages/developer-guide/user-uploads/uploading-files-directly-amazons3-using-uppy.liquid [hunk 10]
```diff
@@ -241,22 +241,22 @@ const createImage = imageUrl => {
...
 method: post
 ---
 {% if context.params.user_id %}
-  {% graphql model, direct_url: context.params.direct_url, user_id: context.params.user_id %}
-    mutation model_create($direct_url: String!, $user_id: ID!) {
-      model_create(model: {
+  {% graphql record, direct_url: context.params.direct_url, user_id: context.params.user_id %}
+    mutation record_create($direct_url: String!, $user_id: ID!) {
+      record_create(record: {
         custom_model_type_name: "modules/direct_s3_upload/uppy"
```
Comment (Slashek) [inline app/views/pages/developer-guide/user-uploads/uploading-files-directly-amazons3-using-uppy.liquid]: this won't run, should be `table` I believe

File: app/views/pages/developer-guide/data-import-export/import.liquid (hunk 7 above)
Comment (diana-lakatos) [reply app/views/pages/developer-guide/data-import-export/import.liquid]: I edited it back, and then we will update it once done. Thank you!

**Comment by diana-lakatos** [comment]
@Slashek Thank you for your review! I edited everything based on your feedback.

**Pull request merged**